charters/ExpressionMatrixService/charter.md

```diff
@@ -0,0 +1,107 @@
+
+* Expression Matrix Service interface supports queries based on:
+    * Bundle identifiers
+        * This query enables selection of specific submissions.
+    * Metadata (*across bundles; potentially provided directly by Query Module*)
+        * These types of filters will enable users to select cells based on their biological characteristics, for example, all kidney cells, or all cells prepared using the 10x v2 assay.
+    * Numerical filtering (*on gene expression values, and also QC metrics*)
+        * These types of filters will, for example, enable users to query for cells according to their expression of key marker genes, or exclude cells with low quality scores.
+
+* Expression Matrix Service API and scalable backend that provides access to
+query interface
+
+* Architecture, software design, and scale testing for scalability
+
+* Data representation
+    * Multiple expression matrix data formats based on _de facto_ standards delivered by the API and the Command Line Interface (CLI)
+    * Expression matrix data format for storage
+    * QC matrix data format delivered by the API and the Command Line Interface (CLI)
+    * QC matrix data format for storage
+
+
+* Extends HCA DCP Command Line Interface (CLI) client to support the Expression Matrix Service
+
+* Documentation
+    * Publish documentation for API and file formats
+    * Publish example code for developers to enable fast/easy adoption of API by analysis and visualization portals
+
+* User Experience
+    * Conduct UX research and design; drive query and API design directly from use cases
+    * Validate design with communities building portals for analysis and
+    visualization
+    * Provide feedback in the form of architecture sessions and written UX
+    reports to the DCP community about the experience of building layered services using the public API(s). (e.g., *can the expression microservice be built using only public Data Store API?*)
+    * Scientific "guardrails" to warn users when executed queries produce data subsets with questionable scientific utility
+        * For example, if a user queries for the expression of only a single gene across all lung cells, this query would trigger a warning that the returned data cannot be effectively normalized
+
+* External dependencies
+    * Dependencies on Data Store, Query Service, and Secondary Analysis to enable and expose data matrices and quality control metrics
+    * Dependency on Query Service for join queries on metadata
+
+## Out-of-scope
+
+* Compound metadata and numerical filtering (*combinations of the In-scope filters*)
+* Data visualization portal or web application for expression matrices
+* Transformative data manipulation; analysis apps or pipelines using expression data (e.g. data normalization)
+* Standards engagement for APIs or data storage formats with the GA4GH Large Scale Genomics work stream prior to basic completion of matrix service implementation and integration of user feedback
```

Comment:
Just a quick check on this statement, I assume the caveat "prior to basic completion of matrix service implementation and integration of user feedback" means when the service is up and running and people are actively using it you will recharter to move standards engagement into scope?

Reply:
We would re-charter and move standards engagement into scope if it aligned with the strategic objectives of the HCA as expressed by guidance from the GG.